sentry-android-replay/src/main/java/io/sentry/android/replay/viewhierarchy/ViewHierarchyNode.kt

```diff
@@ -47,11 +47,11 @@ data class ViewHierarchyNode(
             return actualPosition.intersects(screen.left, screen.top, screen.right, screen.bottom)
         }
 
-        fun adjustAlpha(color: Int): Int {
+        fun Int.toOpaque(): Int {
```

Comment:
`l`: could be simplified to 
```kotlin
fun Int.toOpaque() = this or 0xFF000000.toInt()`
```

Reply:
Yeah I've tried that but for some reason it didn't work on RN somehow... I will add a todo to verify whether it works